src/credential-status/status-list.ts

```diff
@@ -0,0 +1,45 @@
+import * as zlib from 'pako'
+import { cborDecode, cborEncode } from '../cbor'
+import { type AllowedBitsPerEntry, StatusArray, allowedBitsPerEntry } from './status-array'
+
+export interface CborStatusListOptions {
+  statusArray: StatusArray
+  aggregationUri?: string
+}
+
+export class StatusList {
+  static buildCborStatusList(options: CborStatusListOptions): Uint8Array {
+    const compressed = options.statusArray.compress()
+
+    const statusList: Record<string, number | Uint8Array | string> = {
```

Comment:
Might be nice to define some interfaces for this, so we don't have to type it as `Record<string, number  | ....`, also for when we decode the list.